travis.php.ini

```diff
@@ -2,4 +2,4 @@ memory_limit = -1
 display_errors = On
 log_errors = On
 trace_errors = On
-error_log = /var/log/apache2/error.log
```

Comment:
Nice! I had been meaning to look into why this wasn't posting for a while. Going to cherry-pick your improved logging commit and throw up a quick PR so we can get this merged before all the PHP 7.4 stuff.